### PR TITLE
[PR-18309] Actualize Webhook sample payloads

### PR DESCRIPTION
**Pull request by Doimhneacht**
More updates:
- refactored `person_fields` to use substitution
- fixed wrong dashes, missing/extra commas
- added missing fields to payloads
- updated and multi-lined cURL examples

## Demo

https://void-docs.talkable.com/web_hooks.html

## Related Stories

[](https://talkable.atlassian.net/browse/PR-18309)

### CODE REVIEW

File: source/partials/origin_fields.rst [hunk 1]
```diff
@@ -6,22 +6,23 @@
...
 
 *For Purchase:*
 
- * **order_number** - unique identifier of the Purchase
- * **subtotal** - order subtotal for the Purchase
- * **coupon_code** - coupon codes used with the Purchase (separated by ``,`` if multiple)
- * **order_date** - date of the Purchase
+ * **order_number** — unique identifier of the Purchase
+ * **subtotal** — order subtotal for the Purchase
+ * **currency_iso_code** — currency code of the Purchase
```
Comment (dgtv) [inline source/partials/origin_fields.rst:18]:
```suggestion
 * **currency_iso_code** — currency code of the Purchase, defaults to the Site's currency
```

File: source/partials/origin_fields.rst [hunk 2]
```diff
@@ -6,22 +6,23 @@
...
 *For Event:*
 
- * **event_category** - identifier of an action that trigger the Event (e.g. ``app_installed``)
- * **event_number** - unique identifier of the Event within the associated **event_category**
- * **subtotal** - optional monetary attribute of the Event
- * **coupon_code** - coupon codes used with the Event (separated by ``,`` if multiple)
+ * **event_category** — identifier of an action that trigger the Event (e.g. ``app_installed``)
+ * **event_number** — unique identifier of the Event within the associated **event_category**
+ * **subtotal** — optional monetary attribute of the Event
+ * **currency_iso_code** — currency code of the Event, defaults to site currency
```
Comment (dgtv) [inline source/partials/origin_fields.rst:27]:
```suggestion
 * **currency_iso_code** — currency code of the Event, defaults to the Site's currency
```

File: source/partials/person_fields.rst [hunk 3]
```diff
@@ -1,17 +1,18 @@
-* **email** — person’s email address
-* **phone_number** — person’s phone number
-* **first_name** — person’s first name
-* **last_name** — person’s last name
-* **username** — person’s username (optional)
+* **email** — |person|’s email address
```
Comment (dgtv) [inline source/partials/person_fields.rst:1]: Why `|` is used?

File: source/partials/person_fields.rst [hunk 4]
```diff
@@ -1,17 +1,18 @@
...
+* **phone_number** — |person|’s phone number (optional)
+* **first_name** — |person|’s first name
+* **last_name** — |person|’s last name
```
Comment (dgtv) [inline source/partials/person_fields.rst:4]: Also optional

File: source/web_hooks/reward.rst [hunk 5]
```diff
@@ -194,13 +166,30 @@ Reward reason can be of 5 following general types.
...
+       "unsubscribed_at": null,
+       "opted_in_at": null,
+       "phone_opted_in_at": null,
+       "sub_choice": false,
+       "subscribed_at": null,
+       "custom_properties": {},
+       "referral_counts": {"total": 0, "approved": 0, "pending": 0},
+       "is_loyalty_member": false,
+       "loyalty_member": null,
+       "gender": null
```
Comment (dgtv) [inline source/web_hooks/reward.rst:184]: I see `gender` only presents here, but it is deprecated. Can we not include it here?

Comment (Doimhneacht) [reply source/web_hooks/reward.rst:184]: We discussed it with Valeriia, and she asked to document this variable and add it to all payloads.

File: source/partials/person_fields.rst (hunk 3 above)
Comment (Doimhneacht) [reply source/partials/person_fields.rst:1]: It's a substitution. `|person|` should be replaced with whatever is specified in the file where this partial is included.

**Review by dgtv — APPROVED** [review]
LG, thanks!

**Pull request merged**